src/styles/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  width: $product-card-width;
```

Comment:
The variable `$product-card-width` is used here. Ensure this variable is defined in your SCSS files, otherwise, it will cause an error during compilation.